src/EditorFeatures/Core/Shared/Extensions/ITextBufferExtensions.cs

```diff
@@ -43,5 +43,43 @@ internal static bool GetFeatureOnOffOption(this ITextBuffer buffer, PerLanguageO
                 throw ExceptionUtilities.Unreachable;
             }
         }
+
+        internal static bool TryGetOwningWorkspace(this ITextBuffer buffer, out Workspace workspace) =>
```

Comment:
>TryGetOwningWorkspace [](start = 29, length = 21)

The name looks a bit odd. I'd omit `Owning` - the buffer does not own the workspace or vice versa. The buffer is associated with a workspace. `buffer.TryGetWorkspace` seems obvious without any further qualification.

Reply:
:+1: Not sure why i recommended 'Owning'.  It really isn't necessary.